app/js/components/orderEntry/Paginate.jsx

```diff
@@ -11,39 +11,34 @@ const Paginate = ({
   patientId,
 }) => (
   <div className="pagination">
-    { totalPage > 10 ?
-      <div
-        className="page-link prev"
-        value="previous"
-        onClick={() => dispatch(fetchNew(prevPageUrl, patientId))}
-        role="button"
-        tabIndex={0}>
-        <p>❮❮</p>
-      </div>
-      :
-      <div
-        className="page-link-disabled"
-        value="previous"
-        role="button"
-        tabIndex={0}>
-        <p>❮❮</p>
-      </div>
-    }
+    <div
+      className={prevPageUrl ? "page-link prev" : "page-link-disabled"}
+      value="previous"
+      onClick={() => dispatch(fetchNew(prevPageUrl))}
+      role="button"
+      tabIndex={0}>
+      <p>❮❮</p>
+    </div>
+
     <div className="page-link">
-      <p>{prevPageUrl ? Math.floor(prevPageUrl.split('=')[8] / 10) : '1' }</p>
+      <p>{
+        nextPageUrl ? 
+          Math.floor(nextPageUrl.split('=')[nextPageUrl.split('=').length - 1] / 10) 
```

Comment:
What is the meaning of 10?

Reply:
Basically, here we are dividing the `startindex` by 10 in other to get the next page number.

Reply:
Why 10 but not 11 or any other number?

Reply:
10 is used because we need to divide the integer coming from here `(nextPageUrl.split('=')[nextPageUrl.split('=').length - 1]`, to get the next page number. 
like, `30/10`, we get `3` which is the page number.

Reply:
Why not divide by 11, 12, 13, or any other number?

Reply:
We are dividing by 10 because we are having 10 orders in a page and it helps us to display the number of pages.

Reply:
In that case, wouldn't it be clearer to use a properly named constant instead of a magic number?